### PR TITLE
fix csrf_protection configuration for symfony 3

### PR DESCRIPTION
**Pull request by dbu**
otherwise we get 

```
[Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]  
Unrecognized option "csrf_protection" under "framework" 
```

see https://travis-ci.org/doctrine/DoctrinePHPCRBundle/jobs/85928235

check if we get https://github.com/doctrine/DoctrinePHPCRBundle/pull/218 to green with this.

### CODE REVIEW

File: resources/config/dist/framework.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    'default_locale' => 'en',
+    'templating' => array(
+        'engines' => array('twig'),
+    ),
+    'translator' => array(
+        'fallback' => 'en',
+    ),
+);
+
+if (version_compare('2.8.0', Symfony\Component\HttpKernel\Kernel::MAJOR_VERSION, '<=')) {
```
Comment (WouterJ) [inline resources/config/dist/framework.php]: this cannot work as MAJOR_VERSION returns `2` or `3`. Can you change this to:
```php
if (Kernel::VERSION_ID >= 20800) {
    // ...
```

**Comment by WouterJ** [comment]
For 2.0, I want to remove all config files in the component. However, for 1.3 this seems like a correct fix.

File: resources/config/dist/framework.php (hunk 1 above)
Comment (dbu) [inline resources/config/dist/framework.php]: ups. right. sorry

Comment (WouterJ) [inline resources/config/dist/framework.php]: I have to be sorry as well, as it should be < 20800 afaics

**Comment by WouterJ** [comment]
I think this works now?

**Comment by dbu** [comment]
unfortunately no. all builds dont find the dist/monolog.yml file anymore. no idea - do i need to terminate somethinh to change folders in the php config?

**Comment by lsmith77** [comment]
https://travis-ci.org/doctrine/DoctrinePHPCRBundle/jobs/85930111

the error message there sounds strange .. as if its trying to recursively load the monolog.yml

**Comment by lsmith77** [comment]
@dbu did you try this out locally as well?

**Comment by dbu** [comment]
okay, i just pushed a "fix". when i move the ->import('framework.php')
to the end, tests work locally, hope they also work here.

no idea why this is, looks like the relative path is changed when
importing a .php file but not when importing a .yml file. if this and
doctrine bundle get green, i suggest we merge and check if symfony knows
about the problem and report a bug (maybe its a bug in 2.3 that has
since been fixed?)

**Comment by lsmith77** [comment]
+1

**Pull request merged**